api/download.go

```diff
@@ -34,6 +34,9 @@ func download(c *gin.Context) {
 		c.AbortWithStatus(http.StatusBadRequest)
 		return
 	}
+	if err != nil {
+		c.AbortWithStatus(http.StatusExpectationFailed)
```

Comment:
```suggestion
		c.AbortWithStatus(http.StatusExpectationFailed)
		return
```

Reply:
Wait, I think we can just remove the error check ....

Reply:
Oh a second thought this is not necessary, this error is already handled above @alexanderstephan

Reply:
Yes, exactly, I just reverted it. :+1: